api/src/main/java/com/okta/idx/sdk/api/client/IDXAuthenticationWrapper.java

```diff
@@ -382,6 +415,11 @@ public AuthenticationResponse recoverPassword(String username) {
             Map<String, String> authenticatorOptions = remediationOption.getAuthenticatorOptions();
 
             for (Map.Entry<String, String> entry : authenticatorOptions.entrySet()) {
+                if (!entry.getKey().equals(AuthenticatorType.PASSWORD.getValue()) &&
```

Comment:
we only support email and password factor presently. so we dont populate UI with other authenticators if they are available.